test/input_schema_definition.test.ts

```diff
@@ -285,5 +285,163 @@ describe('input_schema.json', () => {
+                            type: 'string',
+                            editor: 'datepicker',
+                            allowAbsolute: false,
+                            allowRelative: true,
+                        },
+                    },
+                })).toBe(true);
+            });
+
+            it('should accept allowAbsolute=true', () => {
+                expect(ajv.validate(inputSchema, {
+                    title: 'Test input schema',
+                    type: 'object',
+                    schemaVersion: 1,
+                    properties: {
+                        myField: {
+                            title: 'Field title',
+                            description: 'My test field',
+                            type: 'string',
+                            editor: 'datepicker',
+                            allowAbsolute: true,
+                        },
+                    },
+                })).toBe(true);
+            });
+
+            it('should accept allowRelative=true', () => {
+                expect(ajv.validate(inputSchema, {
+                    title: 'Test input schema',
+                    type: 'object',
+                    schemaVersion: 1,
+                    properties: {
+                        myField: {
+                            title: 'Field title',
+                            description: 'My test field',
+                            type: 'string',
+                            editor: 'datepicker',
+                            allowRelative: true,
+                        },
+                    },
+                })).toBe(true);
+            });
+
+            it('should accept allowRelative=false', () => {
+                expect(ajv.validate(inputSchema, {
+                    title: 'Test input schema',
+                    type: 'object',
+                    schemaVersion: 1,
+                    properties: {
+                        myField: {
+                            title: 'Field title',
+                            description: 'My test field',
+                            type: 'string',
+                            editor: 'datepicker',
+                            allowRelative: false,
+                        },
+                    },
+                })).toBe(true);
+            });
+
+            it('should not accept allowAbsolute=false', () => {
+                expect(ajv.validate(inputSchema, {
+                    title: 'Test input schema',
+                    type: 'object',
+                    schemaVersion: 1,
+                    properties: {
+                        myField: {
+                            title: 'Field title',
+                            description: 'My test field',
+                            type: 'string',
+                            editor: 'datepicker',
+                            allowAbsolute: false,
+                        },
+                    },
+                })).toBe(false);
```

Comment:
I would rather test if the error message is correct as the logic is not trivial.

Reply:
Added check to error messages